extensions/firefox/content/pdfjschildbootstrap-enabled.js

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2014 Mozilla Foundation
```

Comment:
There is difference in few files for firefox extension and mozilla-central. That includes PdfJsTelemetry.jsm and PdfJs.jsm files -- notice that extension "disables" embedded version of pdf.js by adding PdfJs-stub.

mozilla-central is using pdfjschildbootstrap.js+PdfJs.jsm, but firefox extension only bootstrap.js